CMake/HighFiveConfig.cmake.in

```diff
@@ -1,3 +1,18 @@
+
+# Get HighFive targets
 if(EXISTS "${CMAKE_CURRENT_LIST_DIR}/HighFiveTargets.cmake")
   include("${CMAKE_CURRENT_LIST_DIR}/HighFiveTargets.cmake")
 endif()
+
+# Options when used from external projects. Keep defaults
+if(NOT DEFINED HIGHFIVE_PARALLEL_HDF5)
+  option(HIGHFIVE_PARALLEL_HDF5 "Enable Parallel HDF5 support" @HIGHFIVE_PARALLEL_HDF5@)
+endif()
+if(NOT DEFINED USE_BOOST)
```

Comment:
I am still very much in favour of switching to internally switching to `HIGHFIVE_USE_BOOST` now, and externally allowing both `HIGHFIVE_USE_BOOST` and `USE_BOOST`.